libraries/provider_rack_user.rb

```diff
@@ -26,11 +27,21 @@ def whyrun_supported?
           end
         end
 
+        # default rack password to '*'
+        rack_password = '*'
```

Comment:
Can you change the comment to reflect that isn't a default, but is in-fact, hard-coded :)